src/shipit_signoff/README.md

```diff
@@ -0,0 +1,7 @@
+
+
+    pip install -r requirements.txt
+    cd shipit_signoff
+    swagger-codegen generate -i api.yml -l python-flask 
```

Comment:
It's a work in progress, it was just a set of notes so that other people could get started